src/legacy/core_plugins/kibana/public/discover/__tests__/doc_table/lib/get_sort.js

```diff
@@ -56,7 +56,7 @@ describe('docTable', function() {
       expect(getSort(['lol_nope', 'asc'], indexPattern)).to.eql(defaultSort);
 
       delete indexPattern.timeFieldName;
-      expect(getSort(['non-sortable', 'asc'], indexPattern)).to.eql([{ _score: 'desc' }]);
```

Comment:
Why would `_score` go away as a result of the changes? Wondering the same for all the instances of this change in this file.

Reply:
thx, this is definitely wrong, and I've undone this change